ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml

```diff
@@ -8,13 +8,28 @@ resources:
   '*':
     requests:
       cpu: 1000m
-      memory: 3Gi
+      memory: 1Gi
 tests:
 - artifact_dir: /tmp/artifacts
-  as: sanity
+  as: build
   commands: rpm-ostree --version
```

Comment:
Do we even need this test given that the container build itself is already a prereq for the other tests?

Reply:
I think ci-operator wants us to actually do something with the container image it built.  It's kind of a placeholder for e.g. using this container image for actual cosa builds.